src/__tests__/utils/transform.tsx

```diff
@@ -1,51 +1,120 @@
+import * as Fs from 'fs';
 import * as ts from 'typescript';
-import * as fs from 'fs';
+import { Volume } from 'memfs';
 import * as path from 'path';
+import * as resolveModule from 'resolve';
+import pkg from '../../../package.json';
+import { copy, mkdirp } from './memfs';
 
 const printer = ts.createPrinter();
 
+declare module 'resolve' {
+  export interface SyncOpts extends resolveModule.Opts {
+    /** how to read files synchronously (defaults to fs.readFileSync) */
```

Comment:
is this not covered in `@types/resolve`?

Reply:
Yes, I opened a PR to change it: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41175

Reply:
awesome - for now move it to a resolve.d.ts under the `src/types` folder :)